templates/slurm.conf.erb

```diff
@@ -228,8 +228,9 @@ Licenses=<%= scope['slurm::licenses'] %>
 
 
 
-#MaxJobCount=5000
 #MaxStepCount=40000
+MaxArraySize=<%= scope['slurm::maxarraysize'] %>
```

Comment:
I would suggest to leave the same approach as for the other parameters: if the value is the default one, leave it as comment. In this case: 

```ruby
<% if scope['slurm::maxarraysize'] == 1001 -%>
# MaxArraySize=
<% else -%>
MaxArraySize=<%= scope['slurm::maxarraysize'] %>
<% end -%>
```

Reply:
Good point. I made the parameter optional now with udef as a default value. What do you think about that?

Reply:
yeap, even better. Then the ERB should be adapted to simply `<% if scope['slurm::maxarraysize'] -%>` I guess...